tests/unit/amo/components/TestRatingsByStar.js

```diff
@@ -199,6 +203,37 @@ describe(__filename, () => {
     validateLink(counts.at(4), '1', 'Read all one-star reviews');
   });
 
+  it('adds a `src` query parameter to the review links when available in the location', () => {
+    const src = 'some-src';
+    const grouping = {
```

Comment:
what are these numbers?

Reply:
I believe there are the stars (1 to 5) and then for each "star", there is the number of "votes". It is a bit easier to understand with this screenshot (I hope):

![Screen Shot 2020-03-12 at 14 41 34](https://user-images.githubusercontent.com/217628/76527474-94463d00-646f-11ea-8070-0e10badf9408.png)